ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs

```diff
@@ -0,0 +1,300 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using DD.CBU.Compute.Api.Client;
+using DD.CBU.Compute.Api.Client.Drs;
+using DD.CBU.Compute.Api.Contracts.Drs;
+using DD.CBU.Compute.Api.Contracts.Requests;
+using DD.CBU.Compute.Api.Contracts.Requests.Drs;
+
+namespace Compute.Client.UnitTests.Drs
+{
+	[TestClass]
+	public class ConsistencyGroupAccessorTests : BaseApiClientTestFixture
+	{
+		[TestMethod]
+		public async Task GetConsistencyGroups_ReturnsResponse()
+		{
+			requestsAndResponses.Add(ApiUris.MyAccount, RequestFileResponseType.AsGoodResponse("GetMyAccountDetails.xml"));
```

Comment:
Noticed those mocked calls for the GetMyAccountDetails API in a few tests - don't think its actually needed.

Reply:
Nope, they're not. I've removed them now 👍